libphx/luajit-ffi-gen/src/lua_ffi.rs

```diff
@@ -0,0 +1,306 @@
+use std::{env::VarError, fs::File, io::Write, path::PathBuf};
+
+use crate::{args::AttrArgs, impl_info::ImplInfo};
+
+const LUAJIT_FFI_GEN_DIR_ENV: &str = "LUAJIT_FFI_GEN_DIR";
+const LUAJIT_FFI_GEN_DIR: &str = "../script/ffi";
+// TODO: change to 4 spaces after Lua code refactoring
+const IDENT: &str = "    ";
```

Comment:
Is this TODO completed? I think we can delete this comment now.

Reply:
Yeas, forgot about it. Removed